### ISSUE TITLE
Controller always connects with default profile

### ISSUE THREAD

**Issue by kiler129** [roxgib/anki-contanki#103]
I have a 8BitDo zero 2 configured in macOS mode (Start+A), which is detected by macOS "Dualshock"-compatible controller. I have a custom profile for that controller. However, every time I reconnect the controller, Contanki will always connect it with a default profile for DS4:

<img width="366" alt="image" src="https://github.com/roxgib/anki-contanki/assets/1227834/98bf36e8-a748-416c-a333-3d682a25ff5f">

Basically, I need to go to controller options and always change it to the custom profile.

**Comment by felipemldias** [comment]
I also have a similar issue with an 8bitdo ultimate and an 8bitdo micro.

 I've already set 8bitdo remotes do be discoverable on system settings, but it always recognizes them as multiple different profiles (xbox one, dualshock, and other random unspecified ones)

**Comment by BelgiansWaffles** [comment]
Is there still no update on this? I have the same issuefor both my 8BitDo Zero 2 and DualChosk 4 controllers. They automatically pair as an 8BitDo Pro controller no matter how many times I change it.
